ci-operator/step-registry/ipi/conf/aws/blackholenetwork/ipi-conf-aws-blackholenetwork-commands.sh

```diff
@@ -330,7 +331,23 @@ echo "Subnets : ${subnets}"
 # save stack information to ${SHARED_DIR} for deprovision step
 echo "${STACK_NAME}" >> "${SHARED_DIR}/blackholenetworkstackname"
 
+# Generate working availability zones from the region
+mapfile -t AVAILABILITY_ZONES < <(aws --region "${REGION}" ec2 describe-availability-zones | jq -r '.AvailabilityZones[] | select(.State == "available") | .ZoneName' | sort -u)
+ZONES=("${AVAILABILITY_ZONES[@]:0:${ZONES_COUNT}}")
+ZONES_STR="[ "
+ZONES_STR+=$(join_by , "${ZONES[@]}")
```

Comment:
nit with the existing `master` code you're copying here: this would be simpler as:

```shell
ZONES_STR="[ $(join_by , "${ZONES[@]}") ]"
```